src/Components/ServiceScene/Breakdowns/AddToFiltersButton.tsx

```diff
@@ -24,14 +30,24 @@ import {
 import { FilterOp, NumericFilterOp } from '../../../services/filterTypes';
 
 import { addToFavorites } from '../../../services/favorites';
+import { areArraysEqual } from '../../../services/comparison';
+import { logger } from '../../../services/logger';
 
 export interface AddToFiltersButtonState extends SceneObjectState {
   frame: DataFrame;
   variableName: InterpolatedFilterType;
+  hideExclude?: boolean;
+  isIncluded?: boolean;
+  isExcluded?: boolean;
 }
 
 export class AddFilterEvent extends BusEventBase {
-  constructor(public operator: FilterType | NumericFilterType, public key: string, public value: string) {
+  constructor(
+    public source: 'legend' | 'filterButton' | 'variable',
+    public operator?: FilterType | NumericFilterType,
+    public key?: string,
```

Comment:
```suggestion
    public operator: FilterType | NumericFilterType,
    public key: string,
```
As far as I can see, these should be defined.

Reply:
Fixed in https://github.com/grafana/logs-drilldown/pull/1074/commits/8c3deb8b188c704926b25ba7a014c8c54aa148a1